x-pack/plugins/security_solution/common/detection_engine/utils.ts

```diff
@@ -17,6 +17,14 @@ export const hasNestedEntry = (entries: EntriesArray): boolean => {
   return found.length > 0;
 };
 
+export const hasEqlSequenceQuery = (ruleQuery: string | undefined): boolean => {
+  if (ruleQuery != null) {
+    const parsedQuery = ruleQuery.split(' ');
+    return parsedQuery[0] === 'sequence' && parsedQuery[1] !== 'where';
```

Comment:
@rylnd @rw-access will something like this work, or is it valid syntax to begin a eql query with a leading space?

Reply:
It is valid, or at least valid to pass to the EQL api, so I just trimmed it that should solve any potential edge cases problems

Reply:
might wanna add a test case for other forms of whitespace and multiple, but I think this approach is solid

Reply:
ah yeah good call. new fix should cover that too, thanks